android/app/src/main/java/com/example/android/data/ItemController.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.android.data
+import com.example.android.model.Item
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Header
+
+interface ItemController {
```

Comment:
**question** What is this a controller for?

Reply:
Maybe not an appropriate name.  It is our get items method.

Reply:
I see. In the context of HTTP, I usually think of a "controller" as corresponding to some HTTP server endpoint. I wonder "client" would be more suitable.

Reply:
Thanks.  I'll incorporate.